lang/en/docs/tutorials/materials/specific/passivation-surface-silicon-surface.md

```diff
@@ -0,0 +1,184 @@
+---
+# YAML header
+render_macros: true
+---
+
+# Passivation of Silicon (100) Surface
+
+## Introduction
+
+This tutorial demonstrates how to passivate a reconstructed silicon (100) surface with hydrogen atoms, following the methodology described in the literature.
+
+!!!note "Manuscript"
+    Hansen, U., & Vogl, P.
+    "Hydrogen passivation of silicon surfaces: A classical molecular-dynamics study."
+    Physical Review B, 57(20), 13295–13304. (1998)
+    [DOI: 10.1103/PhysRevB.57.13295](https://doi.org/10.1103/PhysRevB.57.13295){:target='_blank'}.
+
+We will recreate the passivated surface structure shown in Fig. 8:
+
+![Si(100) H-Passivated Surface](/images/tutorials/materials/passivation/passivation_surface_silicon/0-figure-from-manuscript.webp "H-Passivated Silicon (100)")
+
+## 1. Adjust Silicon Surface Structure
+
+### 1.1. Load Base Material
+
+Navigate to [Materials Designer](../../../materials-designer/overview.md) and import the reconstructed Si(100) surface from [Standata](../../../materials-designer/header-menu/input-output/standata-import.md).
+
+![Si(100) Structure](/images/tutorials/materials/passivation/passivation_surface_silicon/1-wave-original-material.webp "Si(100) Structure")
+
+### 1.2. Launch JupyterLite Session
+
+Select the "Advanced > [JupyterLite Transformation](../../../materials-designer/header-menu/advanced/jupyterlite-dialog.md)" menu item to launch the JupyterLite environment.
+
+### 1.3. Open Modified `create_supercell.ipynb` Notebook
+
+Open `create_supercell.ipynb`, select input material as the Si(100) structure, and set the supercell parameters in 1.1.:
+
+```python
+SUPERCELL_MATRIX = [
+    [1, 0, 0], 
+    [0, 1, 0], 
+    [0, 0, 1]
+] 
+
+# or use the scaling factor
+SCALING_FACTOR = None # [3, 3, 1]
+```
+
+Also add to the "Get input materials" cell the following code to adjust the Si atom position:
+
+```python
+from utils.jupyterlite import get_materials
+materials = get_materials(globals())
+material = materials[0]
+
+# Find and adjust Si atom coordinates
+si_atoms_coordinates = [coordinate.value for coordinate in material.basis.coordinates.to_array_of_values_with_ids()]
+si_atoms_sorted = sorted(si_atoms_coordinates, key=lambda x: x[2], reverse=True)
+second_from_top_index = 1
+second_si_atom_coordinate = si_atoms_sorted[second_from_top_index]
+
+print(f"Coordinates of the second Si atom: {second_si_atom_coordinate}")
+adjusted_coordinate = [coord + delta for coord, delta in zip(second_si_atom_coordinate, [0.025, 0, 0.025])]
```

Comment:
Let's reformat this code to make it less wide - otherwise, it's hard to see it online